jsonpath_ng/__init__.py

```diff
@@ -1,6 +1,6 @@
-from .jsonpath import *  # noqa
-from .parser import parse  # noqa
-
-
-# Current package version
-__version__ = '1.5.3'
+from .jsonpath import *  # noqa
+from .parser import parse  # noqa
+
+
+# Current package version
+__version__ = '1.5.3.1'

```

Comment:
Please don't change the package version in this pull request. That will be done the next time a new release is made on PyPI.